moveit_planners/trajopt/include/trajopt_interface/kinematic_terms.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include <moveit/planning_scene/planning_scene.h>
+
+#include <Eigen/Geometry>
+
+using namespace Eigen;
+using namespace std;
+
+namespace trajopt_interface
+{
+Vector3d rotVec(const Matrix3d& m)
+{
+  Quaterniond q;
+  q = m;
+  return Vector3d(q.x(), q.y(), q.z());
+}
+
```

Comment:
add doxygen on what these functions do, there function names are already too terse to be helpful